### PR TITLE
add Angular6 support

### PR DESCRIPTION
**Pull request by unki**
I've updated dependencies for Angular6 - as well as rxjs6 - support.

Also I changed the release number to _2.0.0_ to indicate that previous
versions of this module only support up to Angular 5.
Mentioned that in README.md too.

### CODE REVIEW

**Review by kartikjagdale — COMMENTED** [review]
LGTM :+1:

**Review by kartikjagdale — APPROVED** [review]
Please Can you squash the commits in one commit ?

**Comment by unki** [comment]
@kartikjagdale thx, done

**Review by kartikjagdale — APPROVED** [review]
:+1: LGTM

File: README.md [hunk 1]
```diff
@@ -19,28 +20,41 @@ A really simple, lightweight Angular pipe for converting a date string into a ti
 |546 days+                  | X years ago            |
 
 ## Installation
-```npm install time-ago-pipe --save```
+
+```
+npm install time-ago-pipe --save
+```
```
Comment (thelebdev) [inline README.md:26]: Why change the formatting of the one-liner?

Comment (unki) [reply README.md:26]: Too long ago - can't remember. Probably it was the [markdown linter](https://github.com/markdownlint/markdownlint)

**Review by WilliamRitson — APPROVED** [review]

**Review by jellevdp — APPROVED** [review]

**Pull request closed without merging**